src/main/scala/edu/ie3/powerFactory2psdm/config/ConfigValidator.scala

```diff
@@ -149,4 +160,17 @@ object ConfigValidator {
       )
     )
 
+  private[config] def validateQCharacteristic(
+      qCharacteristic: QCharacteristic
+  ): Try[Unit] = qCharacteristic match {
+    case FixedQCharacteristic => Success(())
+    case DependentQCharacteristic(characteristic) =>
+      try {
+        ReactivePowerCharacteristic.parse(characteristic)
+        Success(())
+      } catch {
+        case exc: Exception => Failure(exc)
+      }
```

Comment:
```suggestion
  ): Try[Unit] = Try(qCharacteristic).flatMap {
    case ConversionConfig.FixedQCharacteristic => Success(())
    case DependentQCharacteristic(characteristic) =>
      Try {
        ReactivePowerCharacteristic.parse(characteristic)
      }.map(_ => ())
  }
```

That would be the Scala'ish way.